### PR TITLE
Feature/hbd-6385

### PR DESCRIPTION
**Pull request by BakJakub**

### CODE REVIEW

File: Handbid_iPad/ViewModels/ChooseOrgAndAuction/ChooseAuctionViewModel.swift [hunk 1]
```diff
@@ -32,8 +32,8 @@ class ChooseAuctionViewModel: ObservableObject, ViewModelTopBarProtocol {
 		setupInitialSelection()
 		setupButtonBindings()
 
-		let user2: UserModel? = try? dataManager.fetchSingle(of: UserModel.self, from: .user)
-		print(user2?.identity)
+		// let user2: UserModel? = try? dataManager.fetchSingle(of: UserModel.self, from: .user)
+		// print(user2?.identity)
```
Comment (mprzypasniak99) [inline Handbid_iPad/ViewModels/ChooseOrgAndAuction/ChooseAuctionViewModel.swift:36]: These two commented lines should be deleted

File: Handbid_iPad/ViewModels/ChooseOrgAndAuction/ChooseOrganizationViewModel.swift [hunk 2]
```diff
@@ -62,14 +62,14 @@ class ChooseOrganizationViewModel: ObservableObject {
 			try? dataManager.create(user, in: .user)
 		}
 
-		let user2: UserModel? = try? dataManager.fetchSingle(of: UserModel.self, from: .user)
-		print(user2?.identity)
+		// let user2: UserModel? = try? dataManager.fetchSingle(of: UserModel.self, from: .user)
+		// print(user2?.identity)
```
Comment (mprzypasniak99) [inline Handbid_iPad/ViewModels/ChooseOrgAndAuction/ChooseOrganizationViewModel.swift:66]: Delete commented lines if they're not used

File: Handbid_iPadTests/Data/WebSocket/HandbidEventDelegateTests.swift
Comment (mprzypasniak99) [inline Handbid_iPadTests/Data/WebSocket/HandbidEventDelegateTests.swift:1]: Tests for `HandbidEventDelegate` are already in the repository, but probably got removed from XCode project on merge. They can be found under `Handbid_iPadTests/Data_Tests/WebSocket/HandbidEventDelegate_Tests.swift`

File: Handbid_iPadTests/Data/WebSocket/HandbidWebSocketFactoryTests.swift
Comment (mprzypasniak99) [inline Handbid_iPadTests/Data/WebSocket/HandbidWebSocketFactoryTests.swift:1]: Tests for `HandbidWebSocketFactory` are already in the repository, but probably got removed from XCode project on merge. They can be found under `Handbid_iPadTests/Data_Tests/WebSocket/HandbidWebSocketFactory_Tests.swift`

Comment (BakJakub) [reply Handbid_iPadTests/Data/WebSocket/HandbidWebSocketFactoryTests.swift:1]: Yes apparently Xcode lost this when merging. The MockWebSocketClient file cannot be found either.  My suggestion is to add the current tests as I want 100% test coverage of everything. We will return to the topic

File: Handbid_iPad/ViewModels/ChooseOrgAndAuction/ChooseAuctionViewModel.swift (hunk 1 above)
Comment (BakJakub) [reply Handbid_iPad/ViewModels/ChooseOrgAndAuction/ChooseAuctionViewModel.swift]: Comments have been deleted

File: Handbid_iPad/ViewModels/ChooseOrgAndAuction/ChooseOrganizationViewModel.swift (hunk 2 above)
Comment (BakJakub) [reply Handbid_iPad/ViewModels/ChooseOrgAndAuction/ChooseOrganizationViewModel.swift]: Comments have been deleted

**Pull request merged**